packages/components/src/disclosure/index.tsx

```diff
@@ -0,0 +1,40 @@
+/**
+ * External dependencies
+ */
+// eslint-disable-next-line no-restricted-imports
+import * as Ariakit from '@ariakit/react';
+
+/**
+ * WordPress dependencies
+ */
+import { forwardRef } from '@wordpress/element';
+
+/**
+ * Internal dependencies
+ */
+import type { DisclosureContentProps } from './types';
+
+/**
+ * Accessible Disclosure component that controls visibility of a section of
+ * content. It follows the WAI-ARIA Disclosure Pattern.
+ *
+ * @see https://ariakit.org/reference/disclosure-content
+ *
+ * The plan is to build own API that accounts for future breaking changes
+ * in Reakit (https://github.com/WordPress/gutenberg/pull/28085).
```

Comment:
Is this comment still relevant, given that we're no longer using Reakit?

Reply:
I meant to rename this one as well but missed it, thanks! I think, for now, the comment still makes sense as we're discussing on how to continue. As other comments on this PR suggest we might want to further develop this component to include other parts of the `Disclosure` components Ariakit provides.

Fixed that Reakit mention via https://github.com/WordPress/gutenberg/pull/55639/commits/a5bef21cb0097764f274076654db41e332e2c248